src/main/python/mmlspark/lightgbm/mixin.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in project root for information.
+
+class LightGBMModelMixin:
+    def saveNativeModel(self, filename, overwrite=True):
+        """
+        Save the booster as string format to a local or WASB remote location.
+        """
+        self._java_obj.saveNativeModel(filename, overwrite)
+
+    def getFeatureImportances(self, importance_type="split"):
+        """
+        Get the feature importances as a list.  The importance_type can be "split" or "gain".
+        """
+        return list(self._java_obj.getFeatureImportances(importance_type))
+
+    def setPredictionCol(self, predictionColName):
+        """
+        Set the prediction column name
+        """
+        self._java_obj.setPredictionCol(predictionColName)
+
+    def setFeaturesCol(self, featuresColName):
```

Comment:
Can you add `setLabelCol` also?

Reply:
note: the new codegen already added these methods, but I refactored the PR to cleanup the python code structure